packages/chromaprint.rb

```diff
@@ -3,41 +3,39 @@
 class Chromaprint < Package
   description 'Chromaprint is a client-side library that implements a custom algorithm for extracting fingerprints from any audio source.'
   homepage 'https://acoustid.org/chromaprint'
-  version '1.5.1-aa67c95'
+  version '1.5.1-aa67c95-1'
   license 'LGPL-2.1'
-  compatibility 'all'
+  compatibility 'x86_64 aarch64 armv7l'
   source_url 'https://github.com/acoustid/chromaprint.git'
   git_hashtag 'aa67c95b9e486884a6d3ee8b0c91207d8c2b0551'
```

Comment:
Newest git hash?

Reply:
Yes, to get it to build against ffmpeg 6...